src/Stryker.Core/Stryker.Core/Reporters/Html/Realtime/Events/SseEvent.cs

```diff
@@ -0,0 +1,16 @@
+﻿using System.Text.Json;
+
+namespace Stryker.Core.Reporters.Html.Realtime.Events;
+
+public class SseEvent<T>
+{
+    public string EventName { get; init; }
+    public T Data { get; init; }
+
+    public override string ToString()
+    {
+        var data = JsonSerializer.Serialize(Data,
+            new JsonSerializerOptions { PropertyNamingPolicy = JsonNamingPolicy.CamelCase });
+        return $"event: {EventName}\ndata:{data}\n\n";
```

Comment:
If you want these fields in your event, why not just make them actual fields in your event?

Reply:
I'm not sure what you mean here 😕

Reply:
Why does SseEvent not include the properties event and data and let the json serializer serialize them, why do you add these fields manually in the ToString.

Reply:
I realize that this is something I should have noticed in mutation testing elements, but I did not :D 
Just wondering if there's a technical reason or if this is just how you designed the event data.

Reply:
> I think we should document the different event's schemas in mutation-testing-elements.

This is something I definitely want to do in the near future 😄 .

> I still feel that abusing ToString is not the best way to go.

Maybe something like `ToPayload()` would fit better?

Reply:
`Serialize` seem fitting to me.

I would suggest using a multiline interpolated string (https://stackoverflow.com/questions/33236592/multiline-c-sharp-interpolated-string-literal) to improve readability.

```
$@"
event: {EventName}
data:{data}
";
```

Or using a StringBuilder. 

The newline between messages should however be added by the sseserver.SendEvent since it's not really a part of the event itself.